lib/recipe.rb

```diff
@@ -0,0 +1,19 @@
+class Recipe
+  attr_accessor :name, :uri, :ingredients, :image, :dietary_info,:url, :source, :id
+
+  def initialize(name, uri, ingredients, image, dietary_info, url, source)
+    @name = name
```

Comment:
You need to verify the arguments to this method otherwise it will fail to split and crash the site.